index.js

```diff
@@ -154,6 +149,10 @@ module.exports = postcss.plugin('postcss-copy-assets', function (copyOpts) {
             result.warn('postcss-copy-assets requires postcss "to" option.');
             return;
         }
+        if (!copyOpts.base) {
+            var str = postCssOpts.to;
+            copyOpts.base = str.substring(0, str.lastIndexOf('/'));
```

Comment:
Can we just use `path.dirname` here and cut it down to a one liner? From what I understand, `path.dirname` is more platform-agnostic.